site/assets/js/tac.js

```diff
@@ -44,13 +44,51 @@
     })
   }, { once: true })
 
-  document.addEventListener('googletagmanager_added', function () {
-    var ask = document.querySelector('#googletagmanagerLine .tarteaucitronAsk')
-    var choiceEvent = 'tarteaucitron.userInterface.respond(document.getElementById(\'googletagmanagerAllowed\'),document.getElementById(\'googletagmanagerAllowed\').checked);'
-    var toggle = document.createRange().createContextualFragment('<div class="form-check form-switch my-2 my-lg-0"><input class="form-check-input ms-0" type="checkbox" id="googletagmanagerAllowed" aria-describedby="tacCLgoogletagmanager" onchange="' + choiceEvent + '"><label class="form-check-label visually-hidden" for="googletagmanagerAllowed">Google Tag Manager</label></div>')
-    ask.innerHTML = ''
-    ask.append(toggle)
-  }, { once: true })
+  document.addEventListener(
+    'googletagmanager_added',
+    function () {
+      var ask = document.querySelector(
+        '#googletagmanagerLine .tarteaucitronAsk'
+      )
+      var choiceEvent =
+        'tarteaucitron.userInterface.respond(document.getElementById(\'googletagmanagerAllowed\'),document.getElementById(\'googletagmanagerAllowed\').checked);'
+      var toggle = document
+        .createRange()
+        .createContextualFragment(
+          '<div class="form-check form-switch my-2 my-lg-0"><input class="form-check-input ms-0" type="checkbox" id="googletagmanagerAllowed" aria-describedby="tacCLgoogletagmanager" onchange="' +
+            choiceEvent +
+            '"><label class="form-check-label visually-hidden" for="googletagmanagerAllowed">Google Tag Manager</label></div>'
+        )
+      ask.innerHTML = ''
+      ask.append(toggle)
+    },
+    { once: true }
+  )
+
+  window.addEventListener(
```

Comment:
This listener already exists at line 41, can't you put the both function content into the same `window.addEventListener('tac.open_panel', function()`?